Maple2.File.Ingest/Mapper/AdditionalEffectMapper.cs

```diff
@@ -37,11 +40,17 @@ public class AdditionalEffectMapper : TypeMapper<AdditionalEffectMetadata> {
                         HpRate: data.ConsumeProperty.hpRate,
                         SpRate: data.ConsumeProperty.spRate),
                     Update: Convert(data),
+                    Status: Convert(data.StatusProperty, data.OffensiveProperty, data.DefensiveProperty),
+                    Offensive: new AdditionalEffectMetadataOffensive(
```

Comment:
can we just combine these 2 into `Status`, seems pointless to have columns for just 1 value